frontend/src/views/Lobby.tsx

```diff
@@ -216,7 +209,14 @@ function LobbyPage() {
       connectUserToRoom(currentRoomId, currentUser.userId);
     }
   }, [socketConnected, connectUserToRoom, currentRoomId, currentUser]);
-
+  
+  // Redirect user to game page if room is active.
+  useEffect(() => {
+    if (active) {
+      history.push('/game');
+    }
+  }, [history, active]);
+  
```

Comment:
Are you seeing the same errors that are in this attached image? It's just deleting spaces, but this should show up as ESLint warnings on your editor (I think we have the same setup). Either way, small nit just to delete those spaces.

<img width="806" alt="ESLint Space Warnings" src="https://user-images.githubusercontent.com/7987279/96641131-1be30c80-12d9-11eb-810e-26c1db92a9a4.png">